### PR TITLE
Sigh/sniff sounds

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
Partial port of: https://github.com/BeeStation/BeeStation-Hornet/pull/7804

https://github.com/tgstation/tgstation/assets/96586172/5ca78b8a-8671-4f25-bf38-8bc980de7c9b

https://github.com/tgstation/tgstation/assets/96586172/71cabdd9-4c60-473f-9a05-7c39512e2ae0
## Why It's Good For The Game
Emotes are a way to express yourself and adding sounds to them makes them more immersive and expressive.
## Changelog
:cl: grungussuss
sound: sniff and sigh emotes now have sounds
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
no sniff soudns this is such a bad idea oh my god

**Comment by Sadboysuss** [comment]
> no sniff soudns this is such a bad idea oh my god

Can you explain why it's a bad idea

**Comment by Graybear98** [comment]
Sighs sounds like villagers from Minecraft

**Comment by DrDiasyl** [comment]
> no sniff soudns this is such a bad idea oh my god

Sniff is funny

**Comment by L0pz** [comment]
I LOVE AUDIO!!!!

**Comment by gorillion** [comment]
can the sigh scale with your character's pitch? i feel like having only two sounds (with what appears to be slight random pitch variance between each play) runs the risk of everyone sounding the same

**Comment by Sadboysuss** [comment]
> can the sigh scale with your character's pitch? i feel like having only two sounds (with what appears to be slight random pitch variance between each play) runs the risk of everyone sounding the same

Out of scope

File: sound/attributions.txt [hunk 1]
```diff
@@ -209,3 +209,10 @@ beaker_pickup.ogg was made by lowering pitch:
 Bottle Tap.wav by alex_alexalex -- https://freesound.org/s/395492/ -- License: Attribution NonCommercial 3.0
 beaker_place.ogg was made by cutting and lowering pitch:
 place glass object.wav by milpower -- https://freesound.org/s/353105/ -- License: Creative Commons 0
+
+{
+female_sigh.ogg
+female_sniff.ogg
+male_sigh.ogg
+male_sniff.ogg
+} - from https://github.com/BeeStation/BeeStation-Hornet/pull/9759
```
Comment (thgvr) [inline sound/attributions.txt:218]: These sounds aren't in the attributed PR.

**Comment by thgvr** [comment]
Honestly, the sigh sounds more like suggestive moan than anything for males. It's very uncomfortable

**Comment by Jacquerel** [comment]
> Honestly, the sigh sounds more like suggestive moan than anything for males. It's very uncomfortable

I'll be honest I have to agree with this.
On balance I think _both_ of these sounds are probably going to be used for more ill than good if merged and as you asked someone to make a quick decision about the PR for you, I will do just that.

**Pull request closed without merging**